bioimageio_chatbot/chatbot.py

```diff
@@ -127,10 +129,10 @@ def create_customer_service(db_path):
     resource_item_stats = f"""Each item contains the following fields: {list(resource_items[0].keys())}\nThe available resource types are: {types}\nSome example tags: {tags}\nHere is an example: {resource_items[0]}"""
     class DocumentRetrievalInput(BaseModel):
         """Input for finding relevant documents from databases."""
-        query: str = Field(description="Query used to retrieve related documents.")
+        query: str = Field(description="Query to be used for retrieving relevant documents at different channel.")
```

Comment:
This should also be a list, no?

Reply:
Maybe create a list of ChannelQuery objects with channel_id and query as property

Reply:
> Maybe create a list of ChannelQuery objects with channel_id and query as property

Add ChannelQuery class, and integrate it as a list in DocumentRetrievalInput